Add a manifest file for GOV.UK PaaS users (#652)
alphagov/govuk-prototype-kit: docs/documentation/publishing-on-govuk-paas.md

# Publishing on the web (GOV.UK PaaS)

We recommend using [GOV.UK PaaS](https://www.cloud.service.gov.uk) to get your prototype online.  It’s simple and fast to deploy new versions as you work.

Once your prototype is on GOV.UK PaaS, other people will be able to access and try your prototype from their own computers or mobile devices.

> **DO NOT** enter real user data in to prototypes hosted on GOV.UK PaaS. If your prototype stores or collects user data, talk to a security professional about appropriate security steps you must take.

A prototype deployed to GOV.UK PaaS is called an `app` - it will have a url like:

`your-prototype.london.cloudapps.digital`

or

`your-prototype.cloudapps.digital`

depending on which region your GOV.UK PaaS account is in.

You can have multiple apps running on GOV.UK PaaS - projects often have several so they can try different ideas out at once.

## 1) Get started with GOV.UK PaaS

> Follow the [GOV.UK PaaS gettin started guide](https://docs.cloud.service.gov.uk/get_started.html#get-started) to:

Reviewer: 🤦‍♀️
```suggestion
Follow the [GOV.UK PaaS getting started guide](https://docs.cloud.service.gov.uk/get_started.html#get-started) to:
```

Reviewer 2: Resolved in 7575565b6d2b9f245bba97ec9c2be83cf005cd34 with the actual page title :D

# Publishing on the web (GOV.UK PaaS)

We recommend using [GOV.UK PaaS](https://www.cloud.service.gov.uk) to get your prototype online.  It’s simple and fast to deploy new versions as you work.

Once your prototype is on GOV.UK PaaS, other people will be able to access and try your prototype from their own computers or mobile devices.

> **DO NOT** enter real user data in to prototypes hosted on GOV.UK PaaS. If your prototype stores or collects user data, talk to a security professional about appropriate security steps you must take.

A prototype deployed to GOV.UK PaaS is called an `app` - it will have a url like:

`your-prototype.london.cloudapps.digital`

or

`your-prototype.cloudapps.digital`

depending on which region your GOV.UK PaaS account is in.

You can have multiple apps running on GOV.UK PaaS - projects often have several so they can try different ideas out at once.

## 1) Get started with GOV.UK PaaS

> The GOV.UK PaaS documentation has [a guide](https://docs.cloud.service.gov.uk/get_started.html#get-started) on how to get started.

Reviewer: ```suggestion
```

Reviewer 2: See eebd11da54c4126c1d4ee3f4e24983f382bf06a5

> You should follow the guide so that you:

Reviewer: ```suggestion
Follow the [GOV.UK PaaS gettin started guide](https://docs.cloud.service.gov.uk/get_started.html#get-started) to:
```

Reviewer 2: See eebd11da54c4126c1d4ee3f4e24983f382bf06a5

> - have downloaded and installed the CF CLI

Reviewer: ```suggestion
- download and install the Cloud Foundry CLI
```

Reviewer 2: See eebd11da54c4126c1d4ee3f4e24983f382bf06a5

> - have used the CF CLI to log in to GOV.UK PaaS

Reviewer: ```suggestion
- log in to GOV.UK PaaS using the Cloud Foundry CLI
```

Reviewer 2: See eebd11da54c4126c1d4ee3f4e24983f382bf06a5

## 2) Choose a name for your app

Think of a name for your app. You need to choose a name that’s unique. The name is used in the url for your prototype. For example the name:

`govuk-payments-prototype`

will create an app at:

`govuk-payments-prototype.london.cloudapps.digital`

or

`govuk-payments-prototype.cloudapps.digital`

depending on which region your GOV.UK PaaS account is in.

## 3) Push your application to GOV.UK PaaS

In the terminal, go to the folder of your prototype and run:

```
cf push --var prototype-name=name-of-your-app \
        --var username=myusername \
        --var password=secret
```

> Replace `name-of-your-app` with your app name from step 4.

Reviewer: ```suggestion
Replace `name-of-your-app` with your app name from step 2.
```

Reviewer 2: 🤦‍♂ ✅
